src/components/Mapbox.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { useRef, useEffect } from 'react';
+import mapboxgl from 'mapbox-gl';
+import { Map, Marker, MapboxOptions, MarkerOptions, LngLatBounds } from 'mapbox-gl';
+import { Result, useSearchState } from '@yext/search-headless-react';
+import 'mapbox-gl/dist/mapbox-gl.css';
```

Comment:
This doesn't load the css multiple times if you have multiple maps right? I feel like it wouldn't but if you could check that would be great!

Reply:
To check this you could try looking at the css that gets loaded onto the page

Reply:
Talked with yen and it looks like the CSS is bundled in

Reply:
it's not bundled in multiple times right?

Reply:
Nope! Just once